utilities/checks.py

```diff
@@ -1,22 +1,43 @@
-from discord.ext.commands import check, Context
-from json import loads
+from discord.ext.commands import check, MissingPermissions
+from functools import wraps
+from discord import Member
 
+from internal.context import Context
 
 def command_enabled(command: str):
```

Comment:
I don't remember, but have we been adding return type annotations on our functions or not?

Reply:
> I don't remember, but have we been adding return type annotations on our functions or not?

I don't see it too often but if you want I can add them.

Reply:
I think I'd rather we maintain consistency within the project itself - if we haven't used it before, might as well not do it now.